elements/lisk-p2p/package.json

```diff
@@ -47,6 +47,7 @@
 		"enableLocalIPs": "sudo ./scripts/enableTestLocalIPs.sh 10 19"
 	},
 	"dependencies": {
+		"@liskhq/lisk-cryptography": "2.1.1",
```

Comment:
As we've done with `lodash`, I'd only include the specific module we use, which in this case is `hash`.

Reply:
I think this is only possible for lodash due to the way they designed the library. But to address @jondubois earlier concern of `sodium-native` being huge, it's actually an optional dependency for `lisk-cryptography` and not installed by default.